src/main/java/com/desiato/puresynth/controllers/AudioController.java

```diff
@@ -19,24 +16,23 @@ public final class AudioController {
     private final AudioService audioService;
 
     @PostMapping("/generate")
-    public ResponseEntity<?> generateAudio(@RequestBody final AudioRequest request) {
+    public ResponseEntity<String> generateAudio(@RequestBody final AudioRequest request) {
+
         try {
+            // Generate the audio file as a byte array
             byte[] audioBytes = audioService.generateSineWaveFile(
```

Comment:
This is leaking a lot of implementation details to the controller. Should be in a service.